item/historical/scripts/util/managers/dataframe.py

```diff
@@ -160,3 +160,34 @@ def create_user_friendly_file(self, df):
         path = "{}/{}".format(cwd, filename)
         df_with_all_countries_data.to_csv(path, index=False)
         print("> UF File saved at: {}".format(cwd))
+
+    def transform_from_uf_to_pf_view(self, df, num_of_none_numeric_columns):
+        '''
```

Comment:
Please use [numpydoc format docstrings](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard), as indicated in our [style guide](https://transportenergy.readthedocs.io/en/latest/reference.html#style-guide).